.changelog/4198.internal.md

```diff
@@ -0,0 +1 @@
+runtime: Deprecate transaction::MethodDispatcher
```

Comment:
Should probably be breaking and say remove instead of deprecate as it was already deprecated before.